packages/experiments/src/components/UnifiedPicker/__snapshots__/UnifiedPicker.test.tsx.snap

```diff
@@ -57,50 +57,53 @@ exports[`UnifiedPicker renders correctly with no items 1`] = `
               &:hover {
                 border-color: #c7e0f4;
               }
-          role="list"
         >
-          <input
-            aria-activedescendant=""
-            aria-autocomplete="both"
-            aria-expanded={true}
-            aria-haspopup="true"
-            aria-owns="suggestion-list"
-            autoCapitalize="off"
-            autoComplete="off"
-            className=
-                ms-BasePicker-input
-                ms-UnifiedPicker-input
-                {
-                  border: none;
-                  flex-grow: 1;
-                  height: 34px;
-                  margin-bottom: 1px;
-                  margin-left: 1px;
-                  margin-right: 1px;
-                  margin-top: 1px;
-                  outline: none;
-                  padding-bottom: 0px;
-                  padding-left: 6px;
-                  padding-right: 6px;
-                  padding-top: 0;
-                }
-                &::-ms-clear {
-                  display: none;
-                }
-            data-lpignore={true}
-            disabled={false}
-            onChange={[Function]}
-            onClick={[Function]}
-            onCompositionEnd={[Function]}
-            onCompositionStart={[Function]}
-            onCompositionUpdate={[Function]}
-            onFocus={[Function]}
-            onInput={[Function]}
-            onKeyDown={[Function]}
-            onPaste={[Function]}
-            role="combobox"
-            value=""
+          <div
```

Comment:
curious what this `<div>` is doing here now..

Reply:
I know it's a test snap, but is it a hint that the code generated poor HTML?

Reply:
ah, the original had
```
<div role="list">
    <input role="combobox" />
</div>
```
but looks like this change turned it into 
```
<div>
    <div role="list" />
    <div role="combobox">
        <input />
    </div>
</div>
```
probably the empty `<div role="list" />` shouldn't be there...

Reply:
There was an empty div if the list had no items.  Not ideal. Fixed in the next update